src/Client.php

```diff
@@ -244,4 +251,11 @@ public function getLogger(): \Psr\Log\LoggerInterface
 
         return $this->logger;
     }
+
+    public function enableDebugLogging(bool $enable = true): Client
```

Comment:
This is a bit of a weird method in that it only has an effect when called before the client first gets created, after which it can still be called but won't really do anything.

With that in mind, this feels more like a configuration option that could be passed in to the constructor - perhaps  as part of the config array?